README.md

```diff
@@ -2,11 +2,6 @@
 
 This project is undergoing a complete rewrite. We're almost there.
 
```

Comment:
The whole Under construction and "this project is undergoing a complete rewrite, we're almost there" can go honestly. We are already there for a good while now :D